linux/disk.go

```diff
@@ -59,7 +59,7 @@ func getDiskType(udInfo disko.UdevInfo) (disko.DiskType, error) {
 		}
 	}
 
-	bd, err := getPartitionsBlockDevice(kname)
+	bd, err := getPartitionsBlockDevice(path.Join("/dev", kname))
```

Comment:
getPartitionsBlockDevice is *supposed* to take either block device name  or path.

Reply:
In my tests this call failed if i didnt give the full path of a disk, meaning for devices name "sda, sdb etc".